testsuite/features/support/twopence_init.rb

```diff
@@ -331,22 +332,19 @@ def client_public_ip(host)
   end
 
   interface = case host
-              when /^sle/, /^opensuse/, /^ssh/, /^ceos/, /^debian9/, /^debian10/, 'server', 'proxy', 'build_host'
-                'eth0'
-              when /^debian11/, /^ubuntu/
+              when /^dblike/, /^debian11/, /^ubuntu/
```

Comment:
I think we should keep debian10 in that list. We still testing it in a BV right? 🤔